css-four-column-layout/styles.css

```diff
@@ -0,0 +1,78 @@
+.container {
+  width: 80%;
+  margin: auto;
+  background-color: #f7f8fa;
+  padding-left: 8rem;
+  padding-right: 8rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.header-row {
```

Comment:
You should make a `row` class with just `display: flex` and make this an additional class you could add along with that class. Try to stick to the plan for layout classes, have `container`, `row`, and `column`, all with just the styling they need for their purpose, keep them reusable.